triage/labels.rst

```diff
@@ -10,405 +10,136 @@ Many labels are shared for both use cases, while some are dedicated
 only to one. Below is a possibly inexhaustive list, but it should get
 you going. For a full list, see `here <https://github.com/python/cpython/issues/labels>`_.
 
-General purpose labels
-======================
 
-:gh-label:`type-behavior`
-    Used for issues/PRs that address unintentional behavior, but do not
-    pose significant security concerns. Generally, bugfixes will be attached
-    to a specific issue where the unintended behavior was first reported.
+.. _general-purpose-labels:
+.. _Type:
```

Comment:
```suggestion
.. _Type:
.. _labels-type:
```

May as well add an actual ref target label here while we're at it, so people aren't tempted to use the legacy ref label or the ambiguous fragment-id only one.